src/components/sbb-teaser/sbb-teaser.tsx

```diff
@@ -17,7 +24,28 @@ import { AccessibilityProperties } from '../../global/interfaces/accessibility-p
 /**
  * Generalized Teaser - for displaying an image, title and paragraph
  */
-export class SbbTeaser implements AccessibilityProperties {
+export class SbbTeaser implements LinkProperties {
+  /**
+   * Teaser variant -
+   * when this is true the text-content will be under the image
+   * otherwise it will be displayed next to the image.
+   */
+  @Prop({ reflect: true }) public isStacked: boolean;
+
+  /**
+   * Heading level of the sbb-title element (e.g. h1-h6).
+   */
+  @Prop() public titleLevel: InterfaceTeaserAttributes['titleLevel'] = '5';
+
+  /** The href value you want to link to. */
+  @Prop() public href!: string | undefined;
```

Comment:
Since it's mandatory, is the undefined type needed?

Reply:
I removed the mandatory flag, as it renders to span if there is no href